tests/ha_cases/run.sh

```diff
@@ -470,60 +560,101 @@ function test_multi_task_reduce_and_restart_worker() {
 
-        run_sql "DROP TABLE if exists $ha_test.ta;" $TIDB_PORT $TIDB_PASSWORD
-        run_sql "DROP TABLE if exists $ha_test.tb;" $TIDB_PORT $TIDB_PASSWORD
-        load_data $MYSQL_PORT1 $MYSQL_PASSWORD1 "a" &
-        load_data $MYSQL_PORT2 $MYSQL_PASSWORD2 "b" &
+    for ((i=0; i < ${#worker_inuse[@]}; i++)); do
+        wk=${worker_inuse[$i]:0-1:1} # get worker id, such as ("1", "4")
+        echo "try to isolate dm-worker$wk"
+        isolate_worker $wk "isolate"
+    done
 
-        run_dm_ctl_with_retry $WORK_DIR "127.0.0.1:$port" \
-            "resume-task test"\
+    run_dm_ctl_with_retry $WORK_DIR "127.0.0.1:$new_leader_port" \
+            "pause-task test"\
             "\"result\": true" 3
 
-        # wait for load data
-        wait
-        sleep 3
-
-        echo "use sync_diff_inspector to check increment data"
-        check_sync_diff $WORK_DIR $cur/conf/diff_config.toml 10
+    run_sql "DROP TABLE if exists $ha_test.ta;" $TIDB_PORT $TIDB_PASSWORD
+    run_sql "DROP TABLE if exists $ha_test.tb;" $TIDB_PORT $TIDB_PASSWORD
+    load_data $MYSQL_PORT1 $MYSQL_PASSWORD1 "a" &
+    load_data $MYSQL_PORT2 $MYSQL_PASSWORD2 "b" &
 
-        isolate_master $idx "disable_isolate"
-        check_rpc_alive $cur/../bin/check_master_online 127.0.0.1:$port
-    done
+    run_dm_ctl_with_retry $WORK_DIR "127.0.0.1:$new_leader_port" \
+        "resume-task test"\
+        "\"result\": true" 3 # wait for load data wait
+    wait
+    sleep 3
 
+    echo "use sync_diff_inspector to check increment data"
+    check_sync_diff $WORK_DIR $cur/conf/diff_config.toml 10
 
-    echo "[$(date)] <<<<<< finish test_isolate_master >>>>>>"
+    echo "[$(date)] <<<<<< finish test_isolate_master_and_worker >>>>>>"
 }
 
 
 function run() {
-    test_join_masters_and_worker               # TICASE-928, 930, 931, 961
-    test_kill_master                           # TICASE-996
-    test_kill_worker                           # TICASE-968, 973, 1002
+    test_join_masters_and_worker               # TICASE-928, 930, 931, 961, 932, 957
+    test_kill_master                           # TICASE-996, 944, 958
+    test_kill_worker                           # TICASE-968, 973, 1002, 975, 969, 972, 974, 970, 971, 976, 978, 988
     test_kill_master_in_sync
     test_kill_worker_in_sync
-    test_standalone_running                    # TICASE-929, 959, 960, 967
+    test_standalone_running                    # TICASE-929, 959, 960, 967, 977, 980
     test_pause_task                            # TICASE-990
-    test_multi_task_reduce_and_restart_worker  # TICASE-968, 994, 995
-    test_isolate_master                        # TICASE-934, 935
-    test_stop_task                             # TICASE-991
+    test_multi_task_reduce_and_restart_worker  # TICASE-968, 994, 995, 964, 966, 979, 981, 982, 985, 986, 989, 993
+    test_isolate_master_and_worker             # TICASE-934, 935, 936, 987, 992, 998
```

Comment:
Is there any comments about `TICASE-999`?

Reply:
I think that test_isolate_master_and_worker already cover it and add comment in 75c8977